src/read.c

```diff
@@ -4712,6 +4728,14 @@ avifResult avifDecoderReset(avifDecoder * decoder)
         }
 #endif // AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP
 
+        // When searching for alpha and gainmap items, it is possible that the data->meta->items array is re-sized in calls to
+        // avifMetaFindOrCreateItem. So it is not safe to store pointers into that array until all the items are either found or
```

Comment:
Nit: "that array" is not clear. Replaced it with "the mainItems array".

Reply:
Done.